bite-size-notes/week-8-c-LQG-stability-smvial.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
I think this is a great review and explanation on the ideas presented in John Doyles paper on LQG stability margins and I like the use of code as part of the explanation. 

I wonder if it wouldn't be too much trouble to add a nyquist or bode plot to visually show the stability margins.

Reply:
I think adding a bode plot would be a fabulous idea.  Unfortunately, I am unable to.  This is my first controls theory class, and they were not covered.  I have received an introduction to them while researching this topic, and have found a textbook co-authored by Steve Brunton (UW ME department) and Nathan Kutz (UW AMATH department) that transitions into robust control theory and teaches bode plots.  It is source 2 of these notes.

Reply:
I noticed that about halfway through the notes I accidentally started referring to Doyle as Boyle... as in Jake Peralta's best friend/partner.  You were just going to let me go with that?! Like letting me go give a speech with something in my teeth.  That's cold.